serverless/serverless
# fix: Ensure to show streaming logs for all lamda versions

## What did you implement



Fixes https://github.com/serverless/serverless/issues/7214

Behind the scenes, log streaming simply fetches from AWS all logs related to a specific log group. For some reason, fetched results were filtered out to show only `$LATEST` version in https://github.com/serverless/serverless/commit/98ec45ce7ed78f5ff7f6341060b0a8588d078180#diff-257addeb65aea4b0585c7e46cae12355R70. 

## How can we verify it
1. Add `provisionedConcurrency: 5` to one of your functions in serverless.yml file. Or you may use the example below.
2. Run `sls deploy` to deploy this change.
3. Run `sls logs -f helloWorld -t` to listen for logs.
4. Make some invocations of your function. The expected result provided by this PR is that the logs are shown. The bug was that it did not show logs run by provisioned lambda instances.
5. ⚠️ After finishing verification, make sure to remove `provisionedConcurrency: 5`. Otherwise, AWS would charge you per second for this provisioned resource.

Here is  `serverless.yml` example you may use for verification:
```
functions:
  helloWorld:
    provisionedConcurrency: 5
    handler: handler.helloWorld
    events:
      - http:
          path: hello-world
          method: get
          cors: true
```
## Todos

<details>
<summary>Useful Scripts</summary>


- `npm run test:ci` --> Run all validation checks on proposed changes
- `npm run lint:updated` --> Lint all the updated files
- `npm run lint:fix` --> Automatically fix lint problems (if possible)
- `npm run prettier-check:updated` --> Check if updated files adhere to Prettier config
- `npm run prettify:updated` --> Prettify all the updated files

</details>



**_Is this ready for review?:_** YES
**_Is it a breaking change?:_** NO
